styles/stylesheet.css

```diff
@@ -0,0 +1,403 @@
+/*colors
+navy #083c60
+green #81b741
+light gray #d3d3d3
+dark gray #3d3d3d
+*/
+
+* {
```

Comment:
I would consider setting the margins for all elements to 0 initially, this ensures any browser styles or overwritten and any margins are only the ones set by you.

Reply:
To add to this, by default chrome adds a margin of 8px to the body for me by default. Setting margins to 0 here would help make sure it takes up all the available space.